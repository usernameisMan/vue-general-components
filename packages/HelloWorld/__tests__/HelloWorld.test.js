'use strict';

const helloWorld = require('..');

describe('HelloWorld', () => {
    it('needs tests');
});
