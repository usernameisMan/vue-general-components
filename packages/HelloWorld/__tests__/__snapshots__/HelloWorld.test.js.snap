// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.test.js has the expected html structure 1`] = `
<div>
  <p
    class="demo-box"
  >
    
    This is 你好 world
  
  </p>
</div>
`;
